src/main/java/bio/terra/service/profile/ProfileService.java

```diff
@@ -108,7 +121,7 @@ public EnumerateBillingProfileModel enumerateProfiles(Integer offset,
     /**
      * Lookup a billing profile by the profile id with auth check. Supports the REST API
      *
-     * @param id the unique idea of this billing profile
+     * @param id   the unique idea of this billing profile
```

Comment:
looks like you introduced some strange spacing here

Reply:
Hmm... something is prettying up the javadoc. I don't think I did it on purpose.